### ISSUE TITLE
'Size' kind of sucks

### ISSUE THREAD

**Issue by CarlQLange** [meteor-utilities/avatar#2]
`(size="large" || "small" || "extra-small")` is sorta crappy. I wonder if there's a good reason for not just using pixel values (or at least overrides for them). Any insight?

**Comment by timothyarmes** [comment]
Yes, this has been a problem for me for ages. I'm working on a solution at the moment.

**Comment by timothyarmes** [comment]
Hi,

I've just created PR #4 for discussion before I merge it into the master branch. If you'd like to try it out you can check out the [sizes](https://github.com/meteor-utilities/avatar/tree/sizes) branch.

In code that is run on both the **server and client** you should now call Avatar.setOptions(), and pass in your custom sizes. e.g.:

``` javascript
Avatar.setOptions({
   customImageProperty: 'profile.avatar',
   fallbackType: 'initials',
   imageSizes: {
     'extra-small': 20,
     'small': 30,
     'medium': 40,
     'nav': 65,
     'large': 80,
     'listing': 90,
     'extra-large': 120
   }
});
```

**Comment by timothyarmes** [comment]
This has now been addressed in version 0.7.15.

**Comment by CarlQLange** [comment]
Awesome, thanks! Sorry I didn't get a chance to check it out beforehand. Can't wait to use it!
